Sources/UnionNotifications/NotificationDialogButton.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  NotificationDialogButton.swift
+//  union-notifications
+//
+//  Created by RJ Kigner on 9/29/25.
+//
+
+import Foundation
+
+import SwiftUI
+import UnionButtons
+
+struct NotificationDialogButton: View {
+    let title: LocalizedStringKey
+    let isEnabled: Bool
+    let height: CGFloat
```

Comment:
Like enabled, framing should never be passed directly into a view. Instead, you should frame the outside of a component with the `.frame(width:height:)` view modifier.